keyboards/prototypist/oceanographer/readme.md

```diff
@@ -0,0 +1,33 @@
+# prototypist/oceanographer
+
+![prototypist/oceanographer](https://i.imgur.com/loDNEHM.png)
+
+*PCB for the Oceanographer keyboard designed by Hedgey.*
+
+* Keyboard Maintainer: [Josh @ Prototypist](https://github.com/Anjheos)
+* Hardware Supported: **Oceanographer**
```

Comment:
Also maybe worth adding a note like this (or feel free to expand on it), because didn't the Oceanographer PCBs from Aeternus ship with an STM32F072 MCU?
```suggestion
* Hardware Supported: Oceanographer PCB with **atmega32u4**
```

Reply:
Without digging into the circumstances around Aeternus, these are the official ones that we're working on with Hedgey. As such the Atmega one right now is the official one.